app/services/github_service.rb

```diff
@@ -1,30 +1,32 @@
 class GithubService
 
-  def list_repos(token)
+  def initialize(token)
+    @token = token
+  end
+
+  def list_repos
     get_url("repos")
-    # response = conn.get("/user/repos?access_token=#{ENV['GITHUB_ACCESS_TOKEN']}")
-    # json = JSON.parse(response.body, symbolize_names: true)
   end
 
-  def list_followers(token)
+  def list_followers
     get_url("followers")
   end
 
-  def list_followings(token)
+  def list_followings
     get_url("following")
   end
 
   private
 
   def get_url(url)
-    response = conn.get("/user/#{url}?access_token=#{ENV['GITHUB_ACCESS_TOKEN']}")
+    response = conn.get("/user/#{url}")
     JSON.parse(response.body, symbolize_names: true)
   end
 
   def conn
     Faraday.new("https://api.github.com") do |req|
       req.adapter Faraday.default_adapter
-      req.params[:key] = ENV['GITHUB_ACCESS_TOKEN']
+      req.headers["Authorization"] = "token #{@token}"
```

Comment:
OAuth2 Token
`curl -H "Authorization: token OAUTH-TOKEN" https://api.github.com`

Reply:
So @token is giving us that OATH TOKEN?

Reply:
Yes. The token is being stored from SearchResults all the way over here.